### ISSUE TITLE
Feature: automaticly update Options on README

### ISSUE THREAD

**Issue by victorperin** [victorperin/qr-scanner-cli#63]
FInd a way to update README documentation based on `src/cli/helpText.js` file.

This improves documentation quality by preventing missleading/wrong/outdated documentation, like we already suffered from.

We can use [@appnest/readme](https://github.com/andreasbm/readme) package.

**Comment by mrcehlo** [comment]
@victorperin I don't think this is possible by any kind of markdown on readme file, but following some opened issues on github with this very same request, I've got this gem that maybe could be of use.

Of course, you would need to setup a hook in order to run the helper, the helper would then update your Readme file based on your class and now you would get an updated readme file.

Not as automatically as I would like, but it is something :)

helper> https://rubygems.org/gems/markdown_helper
repo> https://github.com/BurdetteLamar/markdown_helper

**Comment by victorperin** [comment]
Hello @mrcehlo, thanks for this.
Ok, I need to do some research on it for JS (or maybe could be something to build another package for :smile:).

Tomorrow I'll look into it, If you find sometthing good, please respond here. And If you think this has good enough quality, you can build a PR :)

**Comment by mrcehlo** [comment]
@victorperin 
Thinking about it, perhaps we are looking into it from the wrong perspective.
Wouldn't it be better if we show the path to find the updated options instead of writing it on the Readme?

In this way, instead of showing the options, we would show the command argument to get the options on the terminal:

# > Options
``
qrscanner /?
``
``
qrscanner --help
``
> Find all available options in [helpText.js](https://github.com/victorperin/qr-scanner-cli/blob/master/src/cli/helpText.js)

**Comment by mrcehlo** [comment]
@victorperin 

I took the liberty of updating the readme file with two minor changes:

- Added how to get help from the command line ( I had to look over `meow` docs and also a bit of playing around to get that)
- Added a disclaimer instructing the user that the options described on the readme `may` be outdated.

I believe with this two minor altercations you can get rid of updating this part of the readme for a time now.

**Comment by victorperin** [comment]
Thanks again @mrcehlo, I'll keep this issue opened if anyone find any improvement on that.

**Comment by mrcehlo** [comment]
It was my pleasure contributing! I will also keep an eye here because I'm interested on this topic.

**Comment by victorperin** [comment]
I found this package: [@appnest/readme](https://github.com/andreasbm/readme)

I think this this fits all requisites and can automate this process of documenting.
